src/Jupyter/Extensions.cs

```diff
@@ -103,11 +103,7 @@ public static T WithStackTraceDisplay<T>(this T simulator, IChannel channel)
         simulator.DisableExceptionPrinting();
         simulator.OnException += (exception, stackTrace) =>
         {
-            channel.Display(new DisplayableException
-            {
-                Exception = exception,
-                StackTrace = stackTrace
-            });
+            channel.Display(DisplayableException.Create(exception, stackTrace));
```

Comment:
The `DisplayableException` here will end up [being serialized](https://github.com/microsoft/jupyter-core/blob/4d9233bdede5c0b7135d4e0fc8e4ecb24a54ab4d/src/Engines/BaseEngine.cs#L357) into various formats (text, html and JSON) to be included in a message to sent back to the Jupyter frontend.

Reply:
Jupyter only displays the HTML output, but the Python wrapper uses [JSON results](https://github.com/microsoft/iqsharp/blob/d90578bb70907aa9bdcb4e1234c13f19ea0e27dc/src/Python/qsharp-core/qsharp/clients/iqsharp.py#L388), which is why I can't just simply disable JSON serialization for this type.